packages/table/README.md

```diff
@@ -31,213 +32,144 @@ import '@spectrum-web-components/table/sp-table-row.js';
-        </sp-table-row>
-        <sp-table-row>
-            <sp-table-cell>Row Item Delta</sp-table-cell>
-            <sp-table-cell>Row Item Delta</sp-table-cell>
-            <sp-table-cell>Row Item Delta</sp-table-cell>
-        </sp-table-row>
-        <sp-table-row>
-            <sp-table-cell>Row Item Echo</sp-table-cell>
-            <sp-table-cell>Row Item Echo</sp-table-cell>
-            <sp-table-cell>Row Item Echo</sp-table-cell>
+            <sp-table-cell>Proposal</sp-table-cell>
+            <sp-table-cell>PDF</sp-table-cell>
+            <sp-table-cell>139 KB</sp-table-cell>
+            <sp-table-cell>Published</sp-table-cell>
         </sp-table-row>
     </sp-table-body>
 </sp-table>
 ```
 
-## Selection
-
-To manage selection on an `<sp-table>`, utilise the `selects` attribute on `<sp-table>`. Each `<sp-table-row>` has a `value` attribute which, by default, corresponds to its index in the table, and these `value`s tell `<sp-table>` which `<sp-table-row>`s are selected. The selected items can be manually applied via the `selected` property on the table.
-
-### `selects="single"`
-
-When `selects="single"`, the `<sp-table>` will manage a _single_ selection in the array value of `selected`.
-
-```html
-<sp-table
-    selects="single"
-    selected='["row1"]'
-    onchange="spAlert(this, `Selected: ${JSON.stringify(this.selected)}`)"
->
-    <sp-table-head>
-        <sp-table-head-cell>Column Title</sp-table-head-cell>
-        <sp-table-head-cell>Column Title</sp-table-head-cell>
-        <sp-table-head-cell>Column Title</sp-table-head-cell>
-    </sp-table-head>
-    <sp-table-body>
-        <sp-table-row value="row1">
-            <sp-table-cell>Row Item Alpha</sp-table-cell>
-            <sp-table-cell>Row Item Alpha</sp-table-cell>
-            <sp-table-cell>Row Item Alpha</sp-table-cell>
-        </sp-table-row>
-        <sp-table-row value="row2">
-            <sp-table-cell>Row Item Bravo</sp-table-cell>
-            <sp-table-cell>Row Item Bravo</sp-table-cell>
-            <sp-table-cell>Row Item Bravo</sp-table-cell>
-        </sp-table-row>
-        <sp-table-row value="row3">
-            <sp-table-cell>Row Item Charlie</sp-table-cell>
-            <sp-table-cell>Row Item Charlie</sp-table-cell>
-            <sp-table-cell>Row Item Charlie</sp-table-cell>
-        </sp-table-row>
-        <sp-table-row value="row4">
-            <sp-table-cell>Row Item Delta</sp-table-cell>
-            <sp-table-cell>Row Item Delta</sp-table-cell>
-            <sp-table-cell>Row Item Delta</sp-table-cell>
-        </sp-table-row>
-        <sp-table-row value="row5">
-            <sp-table-cell>Row Item Echo</sp-table-cell>
-            <sp-table-cell>Row Item Echo</sp-table-cell>
-            <sp-table-cell>Row Item Echo</sp-table-cell>
-        </sp-table-row>
-    </sp-table-body>
-</sp-table>
-```
+### Options
 
-### `selects="multiple"`
+<sp-tabs selected="selection" auto label="Table Options">
```

Comment:
```suggestion
<sp-tabs selected="selection" auto label="Table options">
```

Minor nit reminder to use sentence case where possible to demonstrate our content standards.